site/content/docs/5.2/components/badge.md

```diff
@@ -81,6 +103,13 @@ Set a `background-color` with contrasting foreground `color` with [our `.text-bg
 
 Use the `.rounded-pill` utility class to make badges more rounded with a larger `border-radius`.
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
+This component should not be used because it does not exist in the Orange Design System specifications.
+
+Instead, you will have to use the [Tags](https://system.design.orange.com/0c1af118d/p/975c09-tags/b/24dde8) component as described on the Orange Design System website. This component will be added later in Boosted.
```

Comment:
Must be consistent with the other message that I prefer TBH `For other usages, consider using our Boosted [Tags]({{< docsref "/components/tags" >}}) component. You can also refer to the [Tags](https://system.design.orange.com/0c1af118d/p/975c09-tags/b/24dde8) guidelines on the Orange Design System website.`

No need to say that badges will be added later in Boosted because we're not yet 100% sure about it. Otherwise, if we really want to say that, must be everywhere in the callouts of this page, or even at the top of the page as a global information.